docs/rfcs/015-how_we_work/code_reviews.md

```diff
@@ -0,0 +1,23 @@
+# Code Reviews
+
+Code reviews are intended to ensure code changes meet a high standard and to share best practise between team members.
+
+Code reviews should:
+
+- Be performed by a developer who did not write the code.
+- Require approval from:
+   - **one** other developer in the case of project work
+   - **two** developers (one of whom is from a different team) in the case of library changes.
+- Not block release, i.e. changes can be requested in later PRs where that change is not critical to the purpose of the existing PR.
+- Ask for clarification where code or purpose is unclear.
+- Be polite.
+
+Some guidelines:
+
+- Nit picking is okay but should be labelled as such.
+- Developers from any team can comment on any PR that they feel qualified to (it is expected that this happen).
+- The developer who opened a PR should be the one to merge it.
+- Examples of how to structure code or links to resources that clarify are helpful.
+
+
+
```

Comment:
Is there something here around how to take a PR - the thing that comes to mind is "assume good intent". `Polite` is quite subjective.

Maybe something about adding a linting rule is easier than nitpicking code style?